usage/camera/raspicam/v4l2.md

```diff
@@ -0,0 +1,11 @@
+## Using the V4L2 driver
+
+The V4L2 driver provides a Linux standard driver for accessing camera features. This is the driver needed to use the Raspberry Pi camera range as, for example, Webcams.
+
+The V4L2 driver provides a standard API over the firmware based camera system.
+
+
+### Installing the V4L2 driver
+
```

Comment:
Is automatic. It's loaded as a child of VCHIQ, and then checks how many cameras are attached to create the right number of nodes.